layouts/soarian/diff_table.html

```diff
@@ -0,0 +1,214 @@
+<div id="soarian-table-div">
+  <table id="soarian-diff-table">
+    <thead>
+      <tr>
+        <th>DSTU 2 Resource</th>
+        <th colspan="6">Supported Actions</th>
+        <th>R4 Resource</th>
```

Comment:
Try to generate this comparison table through your public sanbox's metadata.

Reply:
yes. Modified to generate comparison table from public sandbox's metadata